_posts/2020-08-24-changelog-129.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #129"
+date: 2020-08-24 04:01:13 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5745 -->
+* Implement [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/) run configuration.
+  So now you can build your WASM library using `wasm-pack` just from the IDE
+
+* Project creation improvements:
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5845 -->
+    * Provide support for custom project templates in the `New Project` dialog
+    
+        <img src="/assets/posts/changelog-129/cargo-generate.png" alt="custom user template screenshot" width="646px"/>
```

Comment:
I don't think that this template would work using cargo-generate. Currently, cargo-generate provides support only for [its special templates](https://github.com/ashleygwilliams/cargo-generate/blob/master/TEMPLATES.md).

As an option, here is updated screenshot.
<img width="884" src="https://user-images.githubusercontent.com/6342851/91054791-89055900-e62c-11ea-9c11-28f0ee42e04d.png">